Dockerfile

```diff
@@ -0,0 +1,114 @@
+#
+#  Globality autogenerated Docker configuration
+#
+#  This file is auto generated with globality-build.
+#  You should not make any changes to this file manually
+#
+#  Any changes made to this file will be overwritten in the
+#  next version of the build.
+#
+#  See: http://github.com/globality-corp/globality-build
+#
+#
+
+# ----------- deps -----------
+# Install from Debian Stretch with modern Python support
+FROM python:slim-stretch as deps
```

Comment:
This tag seems to be [unmaintained](https://hub.docker.com/layers/python/library/python/slim-stretch/images/sha256-12dc3e20ed104ac750612457a853ad333896b7c865498bda9d2913921d9ebc92?context=explore).   Prefer [python:3.7-slim-stretch](https://hub.docker.com/layers/python/library/python/3.7-slim-stretch/images/sha256-e549fc6ae156badf4881c386ea768b65291dd7feff15e48d4f28695ce66eea33?context=explore) ?